textattack/shared/utils/strings.py

```diff
@@ -194,7 +194,7 @@ def zip_flair_result(pred, tag_type="pos-fast"):
     pos_list = []
     for token in tokens:
         word_list.append(token.text)
-        if tag_type == "pos-fast":
+        if "pos" in tag_type:
             pos_list.append(token.annotation_layers["pos"][0]._value)
```

Comment:
Do we need to change this line? I'm not sure what's the difference between `upos-fast` and `pos-fast`...

Reply:
I changed it to `"pos" in tag_type` since it's more general than check if it is `pos-fast`. Yes we still retrieve the POS value using "pos".

Reply:
Ok Thanks!